fbpic/particles/elementary_process/ionization/cuda_methods.py

```diff
@@ -9,8 +9,16 @@
 """
 from numba import cuda
 from scipy.constants import c
-from .inline_functions import get_ionization_probability_cuda, \
-    get_E_amplitude_cuda, copy_ionized_electrons_batch_cuda
```

Comment:
This is a minor change:

The inline functions used to be compiled inside the file `inline_functions`, for both GPU and CPU. This imposed that we give different names to those compiled functions. 

Now they are compiled for GPU in the file `cuda_methods.py` and for CPU in the file `numba_methods.py`. Thus we can keep the same name of the compiled function on GPU and CPU, which reduces divergence between the GPU and CPU code.